README.md

```diff
@@ -214,39 +180,115 @@ models:
     +schema: tiktok_ads
   tiktok_ads_source:
     +schema: tiktok_ads_source
+```
+
+> Provide a blank `+schema: ` to write to the `target_schema` without any suffix.
 
+## (Optional) Step 6: Additional configurations
+<details><summary>Expand for details</summary>
+<br>
+
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
+> IMPORTANT: See the Apple Store [`dbt_project.yml`](https://github.com/fivetran/dbt_apple_store_source/blob/main/dbt_project.yml)  and Google Play [`dbt_project.yml`](https://github.com/fivetran/dbt_google_play_source/blob/main/dbt_project.yml) variable declarations to see the expected names.
+    
+```yml
+vars:
+    <default_source_table_name>_identifier: your_table_name 
 ```
+    
+</details>
+<br>
+
+## (Optional) Step 7: Orchestrate your models with Fivetran Transformations for dbt Core™
+<details><summary>Expand for details</summary>
+<br>
 
-## Database Support
+Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core™ setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
 
-This package has been tested on BigQuery, Snowflake, Redshift, Postgres, and Databricks.
+</details>
+<br>
 
-### Databricks Dispatch Configuration
-dbt `v0.20.0` introduced a new project-level dispatch configuration that enables an "override" setting for all dispatched macros. If you are using a Databricks destination with this package you will need to add the below (or a variation of the below) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+# 🔍 Does this package have dependencies?
+This dbt package is dependent on the following dbt packages. For more information on the below packages, refer to the [dbt hub](https://hub.getdbt.com/) site.
+> **If you have any of these dependent packages in your own `packages.yml` I highly recommend you remove them to ensure there are no package version conflicts.**
+
+# !!! TO DO: DOUBLE CHECK VERSIONS ON RELEASE!!!!
```

Comment:
We should be able to update these now!

Reply:
Also, [dbt-expecations](https://hub.getdbt.com/calogica/dbt_expectations/latest/) is used in google ads and should probably be included here as well.

Reply:
Thanks for catching - I also filed an issue for `google_ads_source` to add that as a dependency there as well!